programs/move_loader_api/src/lib.rs

```diff
@@ -60,13 +77,16 @@ fn arguments_to_locals(args: Vec<TransactionArgument>) -> Vec<Local> {
     locals
 }
 
+fn pubkey_to_address(key: &Pubkey) -> AccountAddress {
+    AccountAddress::new(*to_array_32(key.as_ref()))
```

Comment:
Is this accurate or good enough for now?

Reply:
latter